modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -225,7 +229,11 @@ TEST_P(Test_ONNX_nets, VGG16)
     testONNXModels("vgg16", pb, l1, lInf);
 }
 
+#ifdef BUILD_32BIT
+TEST_P(Test_ONNX_nets, DISABLED_VGG16_bn)  // memory usage >2Gb
+#else
 TEST_P(Test_ONNX_nets, VGG16_bn)
+#endif
```

Comment:
Is it observed for both Linux and Windows configs? In example, there is a test which is skipped only for Win32 builds:
https://github.com/opencv/opencv/blob/10ae0c4364ecf1bebf3a80f7a6570142e4da7632/modules/dnn/test/test_caffe_importer.cpp#L154-L155